watertap/unit_models/tests/test_osmotically_assisted_reverse_osmosis_0D.py

```diff
@@ -494,17 +493,14 @@ def test_CP_calculation_with_kf_fixed(self):
         m.fs.unit.B_comp.fix(B)
         m.fs.unit.structural_parameter.fix(300e-6)
 
-        perm_flow_mass = 0.5
-        perm_mass_frac_NaCl = 0.005
-        perm_mass_frac_H2O = 1 - perm_mass_frac_NaCl
-
-        m.fs.unit.permeate_outlet.flow_mass_phase_comp[0, "Liq", "NaCl"].fix(
-            perm_flow_mass * perm_mass_frac_NaCl
+        m.fs.unit.permeate_inlet.flow_mass_phase_comp[0, "Liq", "H2O"].fix(
+            0.251513390470481
```

Comment:
Are these values the same as previous concentration setting?

Reply:
Yes, just the inlet and outlet are switched.